packages/nocodb/docker/start-litestream.sh

```diff
@@ -11,6 +11,13 @@ fi
 if [ -z "${LITESTREAM_S3_SECRET_ACCESS_KEY}" ] && [ -n "${AWS_SECRET_ACCESS_KEY}" ] ; then
   export LITESTREAM_S3_SECRET_ACCESS_KEY="${AWS_SECRET_ACCESS_KEY}"
 fi
+if [ -z "${LITESTREAM_S3_PATH}" ] && -n [ "${AWS_BUCKET_PATH}" ] ; then
+  export LITESTREAM_S3_PATH="${AWS_BUCKET_PATH}"
+fi
+if [ -z "${LITESTREAM_S3_BUCKET}" ] && [ -n "${AWS_BUCKET}" ] ; then
+  export LITESTREAM_S3_BUCKET="${AWS_BUCKET}"
```

Comment:
I guess, we should also [document](https://docs.nocodb.com/getting-started/self-hosted/environment-variables/) that `AWS_BUCKET_PATH` and `AWS_BUCKET` are deprecated, same as for `AWS_ACCESS_KEY_ID` and `AWS_SECRET_ACCESS_KEY`.

Reply:
Agree and thanks for pointing out.